### ISSUE TITLE
MeanAbsoluteErrorLabels metric causes TypeError Exception

### ISSUE THREAD

**Issue by DieseKartoffel** [ck37/coral-ordinal#10]
When compiling the model with:
```
model.compile(loss = coral.OrdinalCrossEntropy(num_classes = num_labels),
                metrics = [coral.MeanAbsoluteErrorLabels()],
                optimizer=opt)
```
it causes the following exception:

**Comment by gmgeorg** [comment]
Have you successfully run the linked notebook from README end to end?  I don't have any issues with using the metric as documented.

**Comment by gmgeorg** [comment]
@DieseKartoffel Have you encountered this error again?  If yes, can you share a reproducible end to end (toy) example?  I can't reproduce this error either locally in notebook or in unit tests: https://github.com/ck37/coral-ordinal/blob/master/coral_ordinal/tests/test_metrics.py

**Comment by gmgeorg** [comment]
Will close this for now since not reproducible locally or in unit test. @DieseKartoffel Reopen if this is still an issue for you on latest dev branch.
